project/src/main/kotlin/org/ionproject/core/common/interceptors/LoggerInterceptor.kt

```diff
@@ -12,7 +15,8 @@ class LoggerInterceptor : HandlerInterceptorAdapter() {
     override fun preHandle(request: HttpServletRequest, response: HttpServletResponse, handler: Any): Boolean {
         val startTime = System.currentTimeMillis()
         logger.info(
-            "IP:${request.remoteAddr} | Method:${request.method} | Endpoint:${request.requestURI} | Timestamp:${startTime}"
+            "IP:${request.remoteAddr} | Method:${request.method} | Endpoint:${request.requestURI} | Request-Id:${MDC.get(
```

Comment:
Replace Endpoint with URI.
Add the matched template, if available, via the request attribute with key `HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE`

Reply:
Same as before. Not trying to improve logging. Will be done in the future.